src/pages/search-result/search-result.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { Header, VideoThumbnail } from '../../components';
+import { StyledResultsDiv } from './styled';
+import { fetchSearchResults } from '../../api';
+
+class SearchResultPage extends PureComponent {
```

Comment:
Extend from Component and 
Explore the difference between `PureComponent` and `Component`